magento/adobe-stock-integration
# #1618: Duplicated items in Used in Entities filters

### Description (*)

This PR will introduce the fix to the issue wherein duplicate items appear in Used in Entities filters.
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration https://github.com/magento/adobe-stock-integration/issues/1618: Duplicated items in Used in Entities filters
2. ...

### Manual testing scenarios (*)


1. Go to **Content - Media Gallery**
2. Click **Filters**
3. Expand, for example, Used in Pages filter
4. Start typing page into Page Title field
5. There should be no duplicates in the options.
